lib/teaspoon/engine.rb

```diff
@@ -19,7 +19,7 @@ class Engine < ::Rails::Engine
 
       # frameworks
       'jasmine/1.3.1.js',
-      'jasmine/2.0.0.js',
+      'jasmine/2.2.0.js',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.